tests/utils.py

```diff
@@ -1,15 +1,15 @@
 from rest_framework_jwt.compat import get_username
 
 
-def jwt_response_payload_handler(token, user=None, request=None):
+def jwt_response_payload_handler(token, user=None, request=None, **kwargs):
     """
     Returns the response data for both the login and refresh views.
     Override to return a custom response such as including the
     serialized representation of the User.
 
     Example:
 
-    def jwt_response_payload_handler(token, user=None, request=None):
+    def jwt_response_payload_handler(token, user=None, request=None, **kwargs):
         return {
             'token': token,
             'user': UserSerializer(user, context={'request': request}).data
```

Comment:
It's a simple response payload handler used by django-rest-framework-jwt to test its features (actually only one feature - letting people use their own response handlers). It was here a long time ago, I just added the **kwargs param.